meta-test-family.spec

```diff
@@ -1,13 +1,13 @@
 %global framework_name moduleframework
 
-Name:           modularity-testing-framework
+Name:           meta-test-family
```

Comment:
@phracek , I updated copr repo name in docs...  As per spec, I would go with the option number two. And I also would change all occurrences of 'moduleframework', agree? If so, I will proceed.